### DISCUSSION TITLE
looking for maintainers

### DISCUSSION THREAD

**Discussion by fohrloop** [np-8/dash-uploader#122] [Announcements]
dash-uploader has become a quite popular alternative component for uploading files in dash applications. I have my interests currently in other projects, so I would be happy to either share the responsibility or handle the project as whole to hands where the project gets the love and care it deserves.

- Niko Föhr

**Comment by fohrloop** [comment]
Related:
- https://github.com/np-8/dash-uploader/issues/4
- [post on community.plotly.com forum](https://community.plotly.com/t/maintainers-new-home-for-dash-uploader-component/75632)
- [adoptoposs.org listing](https://adoptoposs.org/p/7a6e64b9-b3bf-4c27-8416-66614fb2ddcb)

**Comment by mapix** [comment]
We have heavily used Streamlit in many scientific computing projects, but due to the design limitations of Streamlit itself, we had to switch to Dash. Dash-uploader is a great project and has been heavily used by us in the beginning. We will focus on solving the current security and stability issues. We will submit PRs of relatively better designs that have been tested in production back to Upstream. If there are no other volunteers in the future, we will consider joining the maintenance or taking over after fully understanding the entire community feedback. Thank you very much for creating this excellent component.

**Reply by fohrloop** [reply]
Great to hear! I invited you as a collaborator for the project. Welcome onboard!
